www/js/recommendations.js

```diff
@@ -46,12 +27,12 @@ angular.module('emission.main.recommendations',['emission.services', 'emission.p
 
     $scope.clickSuggestion = function(id) {
```

Comment:
Duplicate code in`CommHelper.getSuggestion()` - one in `clickSuggestion` and one in general at controller load? You should really consolidate to one - copy-pasting code is generally bad because you tend to not keep the two copies in sync. For example, you added the error handling to this call, but forgot to add it to the other one